backend/api/healthz.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/kubeflow/pipelines/backend/api/go_client";
+package api;
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+import "backend/api/pipeline_spec.proto";
+import "backend/api/resource_reference.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+import "backend/api/error.proto";
+
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  responses: {
+    key: "default";
+    value: {
+      schema: {
+        json_schema: {
+          ref: ".api.Status";
+        }
+      }
+    }
+  }
+  // Use bearer token for authorizing access to job service.
+  // Kubernetes client library(https://kubernetes.io/docs/reference/using-api/client-libraries/)
+  // uses bearer token as default for authorization. The section below
+  // ensures security definition object is generated in the swagger definition.
+  // For more details see https://github.com/OAI/OpenAPI-Specification/blob/3.0.0/versions/2.0.md#securityDefinitionsObject
+  security_definitions: {
+    security: {
+      key: "Bearer";
+      value: {
+        type: TYPE_API_KEY;
+        in: IN_HEADER;
+        name: "authorization";
+      }
+    }
+  }
+  security: {
+    security_requirement: {
+      key: "Bearer";
+      value: {};
+    }
+  }
+};
+
+service HealthzService {
+  // Get healthz data.
+  rpc GetHealthz(GetHealthzRequest) returns (GetHealthzResponse) {
+    option (google.api.http) = {
+      get: "/apis/v1beta1/healthz"
+    };
+  }
+}
+
+message GetHealthzRequest {
+    // The healthz request.
+    string id = 1;
+  }
+
+message GetHealthzResponse {
+  // The commit sha.
+  string commit_sha = 1;
+
+  // The tag name
+  string tag_name = 2;
```

Comment:
Shall we remove unused fields first?
Because when accessing healthz api from frontend proxy, the two fields will be renamed to `apiServerCommitSha`. https://github.com/Bobgy/pipelines/blob/6c326ad8eab30387fc18f6b5e8b09bea6d6c52e5/frontend/server/handlers/healthz.ts#L68-L87

We need a horizontal alignment on the best way forward.

Reply:
To be exact, immediate action items:
1. remove the two fields
2. regenerate python client

Reply:
I've just pushed the changes. If I understand correctly `multi_user = 3` is still correct and it should not be changed to `multi_user = 1`.

Reply:
Tests completed successfully so hopefully it is alright.